### PR TITLE
*: Fix go vet shadow

### PR DESCRIPTION
**Pull request by shenli**

### CODE REVIEW

**Comment by coocood** [comment]
LGTM

**Comment by disksing** [comment]
LGTM

File: evaluator/builtin_time.go [hunk 1]
```diff
@@ -511,10 +511,10 @@ func builtinDateArith(args []types.Datum, ctx context.Context) (d types.Datum, e
 		if nodeIntervalIntervalDatum.Kind() == types.KindString {
 			interval = fmt.Sprintf("%v", nodeIntervalIntervalDatum.GetString())
 		} else {
-			ii, err := nodeIntervalIntervalDatum.ToInt64()
-			if err != nil {
```
Comment (zimulala) [inline evaluator/builtin_time.go]: err2 -> err1 ?

**Comment by shenli** [comment]
PTAL @zimulala

**Comment by zimulala** [comment]
LGTM

**Pull request merged**